service/controller/key/key.go

```diff
@@ -348,6 +350,25 @@ func KubeletVolumeSizeFromNode(node v1alpha1.KVMConfigSpecKVMNode) string {
 	return DefaultKubeletDiskSize
 }
 
+func HostVolumesToEnvVar(hostVolumes []v1alpha1.KVMConfigSpecKVMNodeHostVolumes) corev1.EnvVar {
+	var lastElemIndex = len(hostVolumes) - 1
+
+	hostVolumesEnvVar := corev1.EnvVar{
+		Name:  "HOST_DATA_VOLUME_CONFIG",
+		Value: "",
+	}
+
+	for idx, hostVolume := range hostVolumes {
+		hostVolumesEnvVar.Value += fmt.Sprintf("%s:%s", hostVolume.MountTag, hostVolume.HostPath)
```

Comment:
Any chance we have a `:` or `,` character in the `MountTag` or `HostPath`?

Reply:
an host path is a Unix dir and I do not think it's a common way to name dir containing a comma. meanwhile the `MountTag` is an arbitrary name that we pick here. but I do not know if `,` is accepted value. but for how I've designed the k8s-kvm script, we should not have `,` anywhere otherwise it would fail.